rollup/plugins/bundlePlugins.js

```diff
@@ -215,6 +216,19 @@ export function makeExcludeBlockPlugin(type) {
   return plugin;
 }
 
+export function makeShimReplayPlugin() {
```

Comment:
l: Just for consistency, let's rename it to 
```suggestion
export function makeReplayShimPlugin() {
```